models/marts/tests/intermediate/int_model_test_summary.sql

```diff
@@ -32,6 +38,34 @@ count_column_tests as (
     group by 1,2
 ),
 
+count_column_constraints as (
+
+    select
+        node_unique_id as direct_parent_id,
+        name as column_name,
+        case
+            when replace(constraints, '"not_null"', '') != constraints
+            then 1
+            else 0
+        end as constraint_not_null_count,
+        cast((length(constraints) - length(replace(constraints, 'type', ''))) / 4 as int) as constraints_count
```

Comment:
Could we move this logic to Jinja in `base_node_columns.sql`/`get_column_values.sql`?

We can keep `constraints` as a column if people want to drill into the constraints but extracting the ones we are interested in seems better in Jinja rather than doing some clever string operations.

We could give them the names `has_not_null_constraint` and `constraints_count` for example.

Reply:
I've given this a try! Let me know what you think.